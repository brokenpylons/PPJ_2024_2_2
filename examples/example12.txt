var n = 12345;
var digits = 5;
var sum = 0;
for (var i = 1, digits) {
    var digit = n - (n // 10) * 10;
    sum = sum + digit;
    n = n // 10;
}

print(sum);
