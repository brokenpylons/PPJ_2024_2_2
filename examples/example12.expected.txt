real("3") times("*") lparen("(") real("2") minus("-") real("1") rparen(")") 
